src/control/server/main.go

```diff
@@ -123,32 +122,39 @@ func serverMain() error {
 	go grpcServer.Serve(lis)
 	defer grpcServer.GracefulStop()
 
+	if err := mgmtControlServer.nvme.spdk.prep(
+		1024, config.UserName, ""); err != nil {
```

Comment:
To be honest, I always find this line-breaking style weird. Is it idiomatic Go?

Reply:
removed